inference-engine/src/transformations/include/ngraph_ops/framework_node.hpp

```diff
@@ -25,15 +28,27 @@ class TRANSFORMATIONS_API FrameworkNodeAttrs {
 
     const std::string& get_type_name() const { return m_type_name; }
 
-    const std::map<std::string, std::string>& get_attrs() const { return m_attrs; }
+    attrs_t::iterator begin() { return m_attrs.begin(); }
+
+    attrs_t::iterator end() { return m_attrs.end(); }
+
+    attrs_t::const_iterator begin() const { return m_attrs.begin(); }
+
+    attrs_t::const_iterator end() const { return m_attrs.end(); }
+
+    std::string operator[](const std::string & key) { return m_attrs[key]; }
```

Comment:
Looks like this operator can be const

Reply:
No since m_attrs[key] is not constant.

Reply:
1. should not it return `string&`
2. If we just need to take existing value, we can use `m_attrs.at(key)` and make method to be `const`

Reply:
I'll add `string&` in other PR.